docs/src/modules/components/overview/tree-view/TreeViewCommunityOrPro.tsx

```diff
@@ -12,13 +12,13 @@ export default function TreeViewCommunityOrPro() {
           caption={'Community and Pro'}
           title={'Two packages for every need'}
           description={
-            'Start with the free-forever Community version, then upgrade to Pro when you are ready for additional features and components.'
+            'Start with the free-forever Community version, then upgrade to Pro when you&apos;re ready for additional features and components.'
           }
           communityDescription={
             'Free forever under MIT license. Includes core features such as expansion, selection, and label editing.'
           }
           proDescription={
-            'Requires a commercial license. better suited for data rich and enterprise applications, including features such as reordering, lazy loading or virtualization.'
+            'Requires a commercial license. Better suited for data-rich and enterprise applications, with features such as reordering, lazy loading, and virtualization.'
```

Comment:
is the extra comma necessary?

```suggestion
            'Requires a commercial license. Better suited for data-rich and enterprise applications, with features such as reordering, lazy loading and virtualization.'
```